### ISSUE TITLE
How to do only Supervised Training for all Labeled Dataset?

### ISSUE THREAD

**Issue by sarmientoj24** [microsoft/SoftTeacher#35]
My experiments would want to benchmark the following:

- supervised-only training without semi-supervision
- semi-supervised training (with labels and without labels)

Is it possible to do only supervised training here? How do I do it?

**Comment by MendelXu** [comment]
Of course, the script is provided here https://github.com/microsoft/SoftTeacher/blob/main/configs/baseline/faster_rcnn_r50_caffe_fpn_coco_full_720k.py
What you have to do is just replace the config file path in the training command.

**Comment by sarmientoj24** [comment]
Did you mean this script? That is the config file right?
`bash tools/dist_train.sh <CONFIG_FILE_PATH> <NUM_GPUS>`
So what this does is it trains all those in the train2017 and validates on all those validate2017, right?

**Comment by sarmientoj24** [comment]
Just to clarify, the supervised-only training that I am referring to only uses ALL the labeled training data **without the unlabeled training data**

**Comment by MendelXu** [comment]
> Just to clarify, the supervised-only training that I am referring to only uses ALL the labeled training data **without the unlabeled training data**

Yes

**Comment by MendelXu** [comment]
> Did you mean this script? That is the config file right? `bash tools/dist_train.sh <CONFIG_FILE_PATH> <NUM_GPUS>` So what this does is it trains all those in the train2017 and validates on all those validate2017, right?

Yes
